### PR TITLE
Academia page header

### PR DESCRIPTION
**Pull request by tahiya-mongo**
This PR adds the new `Academia` page with header. Since the images and texts are not finalized yet, it is still a work in progress.

### CODE REVIEW

File: src/pages/academia.js [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+        <Layout>
+            <Helmet>
+                <title>{metadata.title}</title>
+            </Helmet>
+            <HeroBanner breadcrumb={academiaBreadcrumbs}>
+                <Header>
+                    <HeaderContent>
+                        <Heading>MongoDB for Academia</Heading>
+
+                        <H3>Educators are more important than ever</H3>
```
Comment (jestapinski) [inline src/pages/academia.js:60]: Screen Shot 2020-05-19 at 10 53 35 AM

File: src/pages/academia.js [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+                        <P>
```
Comment (jestapinski) [inline src/pages/academia.js:62]: font size is good here but line height should be `28px`

File: src/pages/academia.js [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+    const metadata = useSiteMetadata();
+    const academiaBreadcrumbs = [
+        { label: 'Home', target: '/' },
+        { label: 'MongoDB for Academia', target: '/academia' },
+    ];
+
+    return (
+        <Layout>
+            <Helmet>
+                <title>{metadata.title}</title>
```
Comment (jestapinski) [inline src/pages/academia.js:53]:
```suggestion
                <title>Academia - {metadata.title}</title>
```

On other pages we seem to follow the format `pageName - (metadata.title}` so let's do that here

File: src/pages/academia.js [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+
+                        <H3>Educators are more important than ever</H3>
+
+                        <P>
+                            Gain access to teaching resources and materials
+                            created by MongoDB, and inspire the developer of
+                            tomorrow with the database for modern applications.
+                        </P>
+
+                        <StyledButton to="" primary hasArrow={false}>
```
Comment (greggb) [inline src/pages/academia.js:68]: This button will scroll the page down to the form, fyi.

File: src/pages/academia.js [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+    letter-spacing: 3px;
+    text-transform: uppercase;
+`;
+
+const StyledP = styled(P)`
+    line-height: ${lineHeight.default};
+`;
+
+const StyledH3 = styled(H3)`
+    font-size: 32px;
```
Comment (jestapinski) [inline src/pages/academia.js:43]:
```suggestion
    font-size: ${fontSize.xlarge};
```
let's use the value from `theme` here!

File: src/pages/academia.js [hunk 6]
```diff
@@ -0,0 +1,86 @@
...
+
+    return (
+        <Layout>
+            <Helmet>
+                <title>Academia - {metadata.title}</title>
+            </Helmet>
+            <HeroBanner breadcrumb={academiaBreadcrumbs}>
+                <Header>
+                    <HeaderContent>
+                        <Heading>MongoDB for Academia</Heading>
```
Comment (jestapinski) [inline src/pages/academia.js:66]: I might have forgotten to attach this one but let's also just add this style in

Screen Shot 2020-05-19 at 1 13 11 PM

**Pull request merged**